collect_app/src/main/java/org/odk/collect/android/instances/DatabaseInstancesRepository.java

```diff
@@ -25,7 +25,7 @@
 
     @Override
     public Instance getBy(long databaseId) {
-        Cursor c = dao.getInstancesCursorForId("" + databaseId);
```

Comment:
This was such a cheeky line of code 😁. Took me a second to understand what was happening.

Reply:
Yeah, sorry, not my best work. 😄